AnagramSolver.Tests/WordRepositoryTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using AnagramSolver.BusinessLogic.Classes;
+using NUnit.Framework;
+using System;
+
+namespace AnagramSolver.Tests
+{
+    [TestFixture]
+    public class WordRepositoryTests
+    {
+        private WordRepository _wordRepository;
```

Comment:
Make this private variable readonly or const. 

Read a little bit about the differences.

Reply:
If made it const or readonly i get errors all over the place